src/apidata.cc

```diff
@@ -123,8 +123,7 @@ namespace dd
           {
             APIData ad;
             ad.fromRapidJson(jval[cit->name.GetString()]);
-            std::vector<APIData> vad = { ad };
-            add(cit->name.GetString(), vad);
+            add(cit->name.GetString(), ad);
```

Comment:
This is part of #1269 already right ? do we need it here as well ?

Reply:
This is needed for the tests to pass. This is the same modification everywhere, so we can just let git handle it and there should not be any problem.